Sources/ArcGISToolkit/Components/Form/FormInputs/SingleLineTextEntry.swift

```diff
@@ -51,8 +51,10 @@ struct SingleLineTextEntry: View {
         HStack {
             TextField(element.label, text: $text, prompt: Text(element.hint ?? "").foregroundColor(.secondary))
                 .focused($isFocused)
-            if isFocused && !text.isEmpty {
```

Comment:
Turns out that was misaligned with the design, the clear button should be visible whenever the field has a value.